check_docker

```diff
@@ -260,6 +260,9 @@ def check_memory(container, warn, crit, units):
 
         evaluate_numeric_thresholds(container=container, value=usage, warn=warn, crit=crit, units=units, name='memory',
                                     short_name='mem', min=0, max=max)
+    else:
```

Comment:
This breaks the unit tests. Could you update their simulated data like so

      --change--
       json_results = {
            'State': {'StartedAt': now_string},
        }
      --to--
      json_results = {
            'State': {'StartedAt': now_string, 'Status': 'running' },
        }